go/vt/topo/zk2topo/serving_graph.go

```diff
@@ -57,9 +57,6 @@ func (zs *Server) GetSrvKeyspace(ctx context.Context, cell, keyspace string) (*t
 	if err != nil {
 		return nil, err
 	}
-	if len(data) == 0 {
```

Comment:
Can you please comment why you removed this?

(You could mention changes like this in the commit description.)